content/pages/schedule.rst

```diff
@@ -4,80 +4,61 @@ Schedule
+----------------------
+*12:00 - 1:00*
+
+Afternoon Sessions
+------------------
+*1:00 - 3:30*
 
-Session 1
----------
-*10:00 - 11:00*
+Topics covered
+--------------
+
+We are still working out specific topics for this year's fall kickoff however we are planning on discussing at least
+the following:
 
 Introductory Workshop and Navigating Linux
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-First we will:
-
 - Review what DevOps and DevOps Bootcamp are.
 - Cover what DevOps BootCamp will teach you throughout the year.
-- Get you started with your own Linux server that that you can use for hands on
-  demos.
+- Get you started with your own Linux server that that you can use for hands on demos.
 - Get started using that Remote Linux Virtual Machine (VM)!
 
 All you need to do is have a laptop and web browser.
 
-
-Session 2
----------
-*11:00 - 12:00*
-
 Operating Systems, Users, and Files
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-In lesson 2 we will learn:
-
 - What an operating system is.
 - Who (and what) a Linux User can be.
 - What a file is in Linux and how to learn about them.
 
-
-Lunch (Pizza Provided)
-----------------------
-*12:00 - 1:00*
-
-
-Session 3
----------
-*1:00 - 2:30*
-
-Software, Documentation, and Getting on IRC
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-Finally we will cover:
+Software, Documentation, and Getting on IRC & Slack
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 - What software is.
 - How to install software in Linux.
 - How to find documentation for a project.
+- How you communicate with open source projects.
 
-And *lastly* we will get you on Internet Relay Chat (IRC) so you can
-communicate with DevOps Bootcamp teachers and students throughout the year.
-
-
-Session 4
----------
-
-*2:30 - 3:30*
+And finally we will get you on Internet Relay Chat (IRC) and Slack so you can communicate with DevOps Bootcamp teachers
+and students throughout the year.
 
 Industry Panel
 ~~~~~~~~~~~~~~
 
-Network with OSL and CASS professionals and alumni currently working in the
-tech industry. This free-from discussion can answer any questions you have about
-doing DevOps as a career. Panelists include: 
-
-* `Daniel Takamori <https://github.com/pono>`_, System Administrator at the `Apache Software Foundation <http://apache.org/>`_
-* `E. Dunham <https://github.com/edunham>`_, DevOps Engineer at `Mozilla <https://www.mozilla.org/en-US/firefox/new/>`_
-* `Jordan Evans <https://github.com/jordane>`_, Sysadmin at `The Linux Foundation <https://www.linuxfoundation.org>`_
-* `Lance Albertson <https://github.com/ramereth>`_, Director of the `OSU Open Source Lab <http://osuosl.org/>`_
-
- 
+Network with OSL and CASS professionals and alumni currently working in the tech industry. This free-from discussion
+can answer any questions you have about doing DevOps as a career. We are still working choosing panelists but we've had
+professionals from Mozilla, Linux Foundation and Apache Software Foundation in the past.
```

Comment:
In the past, we have had professionals from Mozilla, Linux Foundation, and Apache Software Foundation. This year's panel will be announced in the weeks leading up to the event.